src/DOM/HTML/Indexed/LengthLiteral.purs

```diff
@@ -0,0 +1,12 @@
+module DOM.HTML.Indexed.LengthLiteral where
+
+import Prelude (show, (<>))
+
+data LengthLiteral
+  = Pixels Int
```

Comment:
Yeah, looks like you're right: https://www.w3.org/TR/html5/embedded-content-0.html#attr-dim-width

Although here it says the CSS pixel dimension should be integer.